secondary-clusters.md

```diff
@@ -1,48 +1,126 @@
 
-Set prometheus.server.retention to < 7d, can go down to under 1d if not viewing UI on this cluster
+Kubecost and it’s accompanying Prometheus collect a reduced set of metrics that allow for lower resource/storage usage than a standard Prometheus deployment.
 
-Potentially reduce prometheus.server.persistentVolume.size depending on scale, or outright disable persistent storage
+The following configuration options further reduce resource consumption when not using the Kubecost frontend:
 
-Disable thanos.query and thanos.store components (more info)
+```sh
+--set prometheus.server.retention=2d
+```
+
+Potentially reduce retention even further, metrics are sent to the storage-bucket every 2 hours.
 
-Reduce Prometheus query concurrency with  `prometheus.server.extraArgs.query.max-concurrency=1` if using Kubecost bundled Prometheus or directly in our product if using an external Prometheus/Thanos/Cortex ([values flag](https://github.com/kubecost/cost-analyzer-helm-chart/blob/19908983ed7c8d4ff1d3e62d98537a39ab61bbab/cost-analyzer/values.yaml#L99))
+You can tune prometheus.server.persistentVolume.size depending on scale, or outright disable persistent storage.
 
+## Thanos
 
-For reference, here’s a sample list of helm set command line arguments when running Kubecost on a secondary cluster:
+Disable Thanos components. These are only used for troubleshooting on secondary clusters. See this guide for [troubleshooting via kubectl logs](https://guide.kubecost.com/hc/en-us/articles/4407595964695-Long-Term-Storage#troubleshooting).
 
+> Note: Secondary clusters write to the global storage-bucket via the thanos-sidecar on the prometheus-server pod.
+
+```sh
+--set thanos.compact.enabled=false
+--set thanos.bucket.enabled=false
+--set thanos.query.enabled=false
+--set thanos.queryFrontend.enabled=false
+--set thanos.store.enabled=false
 ```
---set kubecostModel.warmCache=false 
---set kubecostModel.warmSavingsCache=false
---set kubecostModel.etl=false
---set prometheus.server.retention=”2d”
---set prometheus.alertmanager.enabled=false
+
+## Node-Exporter
+
+You can disable node-exporter and the service account if cluster/node rightsizing recommendations are not required.
+
+> Note: node-export must be disabled if there is an existing daemonset.<br>
+> <https://guide.kubecost.com/hc/en-us/articles/4407601830679-Troubleshoot-Install#a-name-node-exporter-a-issue-failedscheduling-kubecost-prometheus-node-exporter>
+
+## Helm Values
+
+For reference, this is a list of the most common settings for efficient secondary clusters:
+
+`secondary-clusters.yaml`
+```yaml
+kubecostModel:
+  warmCache: false
+  warmSavingsCache: false
+  etl: false
+  etlCloudAsset: false
+  maxQueryConcurrency: 1
+global:
+  grafana:
+    enabled: false
+    proxy: false
+prometheus:
+  server:
+    global:
+      external_labels:
+        # cluster_id should be unique for all clusters and the same value as .kubecostProductConfigs.clusterName
+        cluster_id: clusterName
```

Comment:
Should we stick to the same variable we use the in the POC repo and elsewhere in the docs?

Reply:
`kubecostProductConfigs_clusterName`